python/ray/plasma/plasma.py

```diff
@@ -46,6 +47,16 @@ def start_plasma_store(plasma_store_memory=DEFAULT_PLASMA_STORE_MEMORY,
     """
     if use_valgrind and use_profiler:
         raise Exception("Cannot use valgrind and profiler at the same time.")
+
+    if huge_pages_enabled and not (sys.platform == "linux" or
```

Comment:
I am a little worried that other non-linux platforms that do support hugepages will be unnecessarily left out by this check.

Reply:
Which ones are you thinking about?

Reply:
any unix platform that's not linux